CMakeLists.txt

```diff
@@ -363,6 +363,17 @@ if (APPLE AND US_COMPILER_APPLE_CLANG)
   set(CMAKE_MODULE_LINKER_FLAGS "${CMAKE_MODULE_LINKER_FLAGS} -stdlib=libc++ -lc++abi")
 endif()
 
+#-----------------------------------------------------------------------------
+# Set stack as non executable
+#-----------------------------------------------------------------------------
+
+# Add flag to force non executable stack on gcc builds
+# The -Wl,-z noexecstack link option on all gcc builds will
```

Comment:
you can build using clang on linux. does this linker option work for clang as well as gcc?